docs/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,21 @@
+export const Footer = () => {
+  return (
+    <footer
+      className={`not-prose sticky top-0 z-50 
+        flex flex-wrap items-center justify-between
+        bg-white
+        py-5 px-4 shadow-md
+        shadow-slate-900/5 
+        dark:bg-slate-900/95 dark:shadow-none
+        sm:px-6 lg:px-8`}
+      aria-label="footer"
+    >
+      <p>
+        © 2022{' '}
```

Comment:
`new Date().getFullYear()`?

Reply:
I'm not a lawyer, but I don't think it is appropriate to have the copyright date update automatically like this.

Reply:
Let's leave it for the next round. Just wanted an excuse to put "ExecOnline" somewhere. We're going to have a variety of GPDR and State of California privacy law stuff to embed if this is to become a public site

Reply:
I'll be working on the GPDR and CCPA updates for the platform soon, so I'll try to take note of everything I learn along the way to make that process smoother once we reach that point with this docsite 👍🏻